### DISCUSSION TITLE
Question about phase images.

### DISCUSSION THREAD

**Discussion by lijiaxiang63** [kschan0214/sepia#65] [Q&A] [answered]
Hi, I am using sepia for my own data. After converting dicom files to nifty files, the phase images look very weird. The tools I use for conversion is dcm2niix. The image intensity range is also not in -pi to pi.

<img width="922" alt="Screenshot 2023-06-01 at 15 07 59" src="https://github.com/kschan0214/sepia/assets/20365998/47e5a001-6bf6-4ad2-a7aa-9d091d726d92">

Does anyone know how to deal with this problem? I am absolutely new to QSM. Thank you!

I also attached the first echo of the data below.

[sub-0001_e1.zip](https://github.com/kschan0214/sepia/files/11622230/sub-0001_e1.zip)

**Comment by kschan0214** [comment, marked as answer]
Hi, 

The values in the nifti images are the same as stored in the DICOM images. Quite often scaling is applied on the phase data so that the data is stored in integer type in the DICOM (the scaling factor could be different for different vendors and scanners). Therefore, rescaling is usually required to bring the the phase data to the wrapped radian range (which will be taking care of in the SEPIA processing pipeline).

For the data acquired with GE scanners, it is quite common to see the discontinuity between the odd and even number slices after the dcm2niix conversion. If you have the real and imaginary part of the image, you can correct the interslice phase polarity difference using the utility function in SEPIA:
<img width="1120" alt="Screenshot 2023-06-01 at 13 51 18" src="https://github.com/kschan0214/sepia/assets/39571502/0cfc14e8-3b68-49dd-8e4e-e13d2f86e12f">

If you only have the phase images, you can try the following script:
[ge_nifti_phase_corr.m.zip](https://github.com/kschan0214/sepia/files/11624750/ge_nifti_phase_corr.m.zip)
You should be able to see the discontinuity disappeared after the correction (and the output data is in radian).
<img width="1146" alt="Screenshot 2023-06-01 at 13 55 20" src="https://github.com/kschan0214/sepia/assets/39571502/40151446-7338-4ed6-a6c8-2a93d5aaa75b">

**Reply by lijiaxiang63** [reply]
Thank you so much for your help!
